### PR TITLE
Feature/hoverfly orcid

### PR DESCRIPTION
**Pull request by garyluu**
Hoverfly mock sandbox ORCID in the unit test. 
Swapped to Java HttpClient to not deal with closing.

Integration test PR will come later.

Memory issues still present. gonna figure it out in another PR

### CODE REVIEW

File: dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java [hunk 1]
```diff
@@ -105,43 +104,41 @@ public static String getOrcidWorkString(Entry e, Optional<Version> optionalVersi
...
-        postRequest.addHeader(HttpHeaders.AUTHORIZATION, "Bearer " + token);
-        StringEntity workEntity = new StringEntity(workString);
-        postRequest.setEntity(workEntity);
-        try (CloseableHttpClient httpClient = HttpClientBuilder.create().build();) {
-            return httpClient.execute(postRequest);
+    public static String getPutCodeFromLocation(HttpResponse httpResponse) {
+        Optional<String> location = httpResponse.headers().firstValue("Location");
+        URI uri;
+        try {
+            uri = new URI(location.get());
```
Comment (coverbeck) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java:136]: Is it safe to assume location is never empty?

**Review by coverbeck — APPROVED** [review]

File: dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java (hunk 1 above)
Comment (garyluu) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java:136]: i think it's fairly safe. an unsuccessful response would've thrown an exception before this

**Review by denis-yuen — APPROVED** [review]

File: dockstore-integration-testing/pom.xml [hunk 2]
```diff
@@ -69,7 +69,6 @@
             <groupId>io.dockstore</groupId>
             <artifactId>dockstore-common</artifactId>
             <version>1.11.0-alpha.2-SNAPSHOT</version>
-            <!-- looks like these are excluded because they bring in duplicate liquibase.build.properties -->
```
[the remark this answers is not in the document]
Comment (garyluu) [reply dockstore-integration-testing/pom.xml:72]: Oops, removed it locally when getting intellij to run tests. must've commited that by accident

**Review by denis-yuen — COMMENTED** [review]
Sonarcloud warnings seem simple and worth handling

**Review by NatalieEO — APPROVED** [review]

**Pull request merged**